apps/precinct-scanner/src/App.test.tsx

```diff
@@ -369,6 +375,7 @@ test('admin and pollworker configuration', async () => {
 test('voter can cast a ballot that scans successfully ', async () => {
   const card = new MemoryCard()
   const hardware = await MemoryHardware.buildStandard()
+  await hardware.setPrinterConnected(false)
```

Comment:
No `act` here?

Reply:
this is before we render the component so I don't think we need an act yet as there is nothing to cause unintended side effects to.